context.go

```diff
@@ -29,12 +30,15 @@ type PhaseResponse struct {
 }
 
 type RequestContext struct {
-	Scheme      string
-	Authority   string
-	Method      string
-	Path        string
-	RequestId   string
+	Scheme    string
+	Authority string
+	Method    string
+	Path      string
+	RequestId string
+
 	Headers     map[string][]string
+	ByteHeaders map[string][]byte
```

Comment:
`RawHeaders` to match envoy's `raw_values`?

Reply:
Updated